Gemfile

```diff
@@ -77,6 +77,9 @@ gem "sqlite3",                        "~>1.3.0",       :require => false
 gem "sys-filesystem",                 "~>1.2.0"
 gem "terminal",                                        :require => false
 
+# temporary inclusion till we can get ConvertTimeToEoTime patch into upstream
+gem "et-orbi"
```

Comment:
Can this live next to the rufus-scheduler gem (or at least a comment to show they are related?)